File: src/buldringno/Entities/Area.cs
using System;
using System.Collections.Generic;

namespace BuldringNo.Entities
{
    public class Area : IEntityBase
    {
        public Area()
        {
            Boulders = new List<Boulder>();
        }
        public int Id { get; set; }
        public string Title { get; set; }
        public string DescriptionMain { get; set; }
        public string DescriptionSecondary { get; set; }
        public string Parking { get; set; }
        public string ApproachTime { get; set; }
        public DateTime DateCreated { get; set; }
        public virtual ICollection<Boulder> Boulders { get; set; }
    }
}

File: src/buldringno/wwwroot/lib/spa/core/domain/area.js
"use strict";
var Area = (function () {
    function Area(id, title, descriptionMain, descriptionSecondary, parking, approachTime, dateCreated, totalBoulders) {
        this.Id = id;
        this.Title = title;
        this.DescriptionMain = descriptionMain;
        this.DescriptionSecondary = descriptionSecondary;
        this.Parking = parking;
        this.ApproachTime = approachTime;
        this.DateCreated = dateCreated;
        this.TotalBoulders = totalBoulders;
    }
    return Area;
}());
exports.Area = Area;
